code/modules/power/smes.dm

```diff
@@ -15,6 +15,9 @@ TYPEINFO(/obj/machinery/power/smes/magical)
 		charge = INFINITY
 		..()
 
+	set_broken()
```

Comment:
Might want to make sure the `hardened` APCs in listening posts etc can't break too

Reply:
atomized to https://github.com/goonstation/goonstation/pull/22894